### ISSUE TITLE
Artnet neopixel sd 

### ISSUE THREAD

**Issue by Lotsanoiz** [natcl/Artnet#47]
How and where can i enter the start record, stop record and play value's  in the neopixel sd sketch ?

**Comment by MathieuMH** [comment]
@Lotsanoiz SD card support is not covered in this repository.
You can always make your own implementation, and add it to the examples once you got it working.

**Comment by natcl** [comment]
Sd recording is triggered via artnet by setting certain values, is that what you mean ?


--
lecaude.com 
studioimaginaire.com

**Comment by Lotsanoiz** [comment]
Yes, i thought maybe it was possible to enter these value whitin serial monitor to start en stop recording.
I use this sketch now : https://github.com/tangophi/Artnet_DMX_SD_Card
but sometimes it fails for no reason.

**Comment by MathieuMH** [comment]
@Lotsanoiz I guess you better open an issue at that repository. Here only the Art-Net lib is supported.
